### PR TITLE
fix(json-schema-form): use items instead of prefixItems

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] Updated Storybook documents
- [ ] Tested with console(if usages are changed) 

### Description

SSIA

image

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: src/inputs/text-editor/PTextEditor.vue [hunk 1]
```diff
@@ -204,7 +204,7 @@ export default defineComponent<Props>({
             state.cmInstance = editor;
         };
 
-        watch([() => state.textareaRef, () => props.code, () => props.disableAutoReformat], async ([textareaRef, code]) => {
```
Comment (minseolee) [inline src/inputs/text-editor/PTextEditor.vue:207]: Why was it async 🤔...
A hawk's eye!

**Review by minseolee — APPROVED** [review]
LGTM!

**Pull request merged**